framework/validators/EachValidator.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\validators;
+
+use yii\base\InvalidConfigException;
+use Yii;
+use yii\base\Model;
+
+/**
+ * EachValidator serves validation of the array attributes.
+ * It perform validation of each array element using any other validator specified by [[rule]].
+ *
+ * ~~~php
+ * class MyModel extends Model
+ * {
+ *     public $arrayAttribute = [];
+ *
+ *     public function rules()
+ *     {
+ *         return [
+ *             ['arrayAttribute', 'each', 'rule' => ['trim']],
+ *             ['arrayAttribute', 'each', 'rule' => ['integer']],
+ *         ]
+ *     }
+ * }
+ * ~~~
+ *
+ * Note: this validator will not work with validation declared via model inline method.
```

Comment:
It can not be created.
Inline validator from model method created here:
https://github.com/yiisoft/yii2/blob/master/framework/validators/Validator.php#L189

While creating slave validator I simply don't have an owner model instance at hand, so I have to pass a dummy model instead.
So if someone pass inline method name into a `rule` it will simply fail with message 'unable to find class "myInlineValidator"'.